feature-libs/product-configurator/rulebased/styles/_configurator-overview-attribute.scss

```diff
@@ -6,6 +6,7 @@
 
   .cx-attribute-value {
     width: 100%;
+    font-weight: 700;
```

Comment:
We need to document this change in  `/docs/migration/css-changes-in-version-4.md`

Reply:
Done